CHANGELOG.md

```diff
@@ -1,12 +1,5 @@
 ### Changelog
 
-## VERSION 6.5.3
```

Comment:
Why have these versions been removed? These were publicly available versions of the SDK, yes?